src/node_crypto.cc

```diff
@@ -3648,6 +3648,45 @@ void Sign::SignFinal(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(rc);
 }
 
+typedef enum {
```

Comment:
No need to `typedef`, `enum ParsePublicKeyResult` works too :)

Reply:
Thanks, Anna :)